// angular-froala-wysiwyg/__ivy_ngcc__/index.js
import { NgModule } from '@angular/core';
import { FroalaEditorModule } from './editor';
import { FroalaViewModule } from './view';
import * as ɵngcc0 from '@angular/core';
import * as ɵngcc1 from './editor/editor.module';
import * as ɵngcc2 from './view/view.module';
export { FroalaEditorDirective, FroalaEditorModule } from './editor';
export { FroalaViewDirective, FroalaViewModule } from './view';
var MODULES = [
    FroalaEditorModule,
    FroalaViewModule
];
var FERootModule = /** @class */ (function () {
    function FERootModule() {
    }
FERootModule.ɵmod = ɵngcc0.ɵɵdefineNgModule({ type: FERootModule });
FERootModule.ɵinj = ɵngcc0.ɵɵdefineInjector({ factory: function FERootModule_Factory(t) { return new (t || FERootModule)(); }, imports: [[
            FroalaEditorModule.forRoot(),
            FroalaViewModule.forRoot()
        ], FroalaEditorModule,
        FroalaViewModule] });
(function () { (typeof ngJitMode === "undefined" || ngJitMode) && ɵngcc0.ɵɵsetNgModuleScope(FERootModule, { imports: [ɵngcc1.FroalaEditorModule, ɵngcc2.FroalaViewModule], exports: [FroalaEditorModule,
        FroalaViewModule] }); })();
/*@__PURE__*/ (function () { ɵngcc0.ɵsetClassMetadata(FERootModule, [{
        type: NgModule,
        args: [{
                imports: [
                    FroalaEditorModule.forRoot(),
                    FroalaViewModule.forRoot()
                ],
                exports: MODULES
            }]
    }], function () { return []; }, null); })();
    return FERootModule;
}());
export { FERootModule };
