// angular-froala-wysiwyg/__ivy_ngcc__/view/view.module.js
import { NgModule } from '@angular/core';
import { FroalaViewDirective } from './view.directive';
import * as ɵngcc0 from '@angular/core';
var FroalaViewModule = /** @class */ (function () {
    function FroalaViewModule() {
    }
    FroalaViewModule.forRoot = function () {
        return { ngModule: FroalaViewModule, providers: [] };
    };
FroalaViewModule.ɵmod = ɵngcc0.ɵɵdefineNgModule({ type: FroalaViewModule });
FroalaViewModule.ɵinj = ɵngcc0.ɵɵdefineInjector({ factory: function FroalaViewModule_Factory(t) { return new (t || FroalaViewModule)(); } });
(function () { (typeof ngJitMode === "undefined" || ngJitMode) && ɵngcc0.ɵɵsetNgModuleScope(FroalaViewModule, { declarations: function () { return [FroalaViewDirective]; }, exports: function () { return [FroalaViewDirective]; } }); })();
/*@__PURE__*/ (function () { ɵngcc0.ɵsetClassMetadata(FroalaViewModule, [{
        type: NgModule,
        args: [{
                declarations: [FroalaViewDirective],
                exports: [FroalaViewDirective]
            }]
    }], function () { return []; }, null); })();
    return FroalaViewModule;
}());
export { FroalaViewModule };
